src/frontend/Komponenter/Behandling/Inngangsvilkår/vilkår.ts

```diff
@@ -28,9 +28,16 @@ export interface IVilkårGrunnlag {
     dokumentasjon?: IDokumentasjonGrunnlag;
     harAvsluttetArbeidsforhold: boolean;
     harKontantstøttePerioder?: boolean;
```

Comment:
Siden man endte opp med å måtte beholde `harKontantstøttePerioder` på grunn av historisk visning, kan man legge til en kommentar på det? `// må beholdes for å sikre korrekt historisk visning`